packages/app/components/Queue/StudentInfoCard.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useMemo } from "react";
+import styled from "styled-components";
+import { Card, Row, Col, Avatar, Button } from "antd";
+import { UserOutlined } from "@ant-design/icons";
+
+const HelpCard = styled(Card)`
+  margin-bottom: 16px;
+`;
+
+const Email = styled.div`
+  font-style: normal;
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 20px;
+  color: #bfbfbf;
+`;
+
+const Name = styled.div`
+  font-weight: 500;
+  font-size: 14px;
+`;
+
+const HeadingText = styled.div`
+  color: #bfbfbf;
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 22px;
+  font-variant: small-caps;
+`;
+
+const BodyText = styled.div`
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 20px;
+  color: #595959;
+  margin-bottom: 16px;
+`;
+
+const HalfButton = styled(Button)`
+  width: 48%;
+`;
+
+const FinishButton = styled(Button)`
+  margin-top: 12px;
+`;
+
+const StudentInfoCard = () => {
+  return useMemo(() => {
+    return (
+      <HelpCard
+        headStyle={{ padding: "0 16px" }}
+        bodyStyle={{ padding: "16px" }}
+        title={
+          <Row justify="space-between">
+            <Name>Alex Takayama</Name>
+            <Email>takayama.a@northeastern.edu</Email>
```

Comment:
Is the email necessary here? If not, it might work to put the finish helping button there to save some space

Reply:
Not sure if the email is necessary, but I like being able to see the student's email on waitwhile so that I can make sure I'm contacting the right student. I think it might be easier for the user if all the buttons are in one place, I tried putting the finish button in the header before and it felt out of place.

Reply:
That makes a lot of sense. We should leave the email as it is then